content/zh-cn/blog/_posts/2022-12-12-kubernetes-release-artifact-signing.md

```diff
@@ -0,0 +1,215 @@
+---
+layout: blog
+title: "Kubernetes 1.26: 我们现在正在对二进制发布工件进行签名!"
+date: 2022-12-12
+slug: kubernetes-release-artifact-signing
+---
+<!--
+layout: blog
+title: "Kubernetes 1.26: We're now signing our binary release artifacts!"
+date: 2022-12-12
+slug: kubernetes-release-artifact-signing
+-->
+
+<!--
+**Author:** Sascha Grunert
+-->
+**作者：** Sascha Grunert
+
+**译者：** XiaoYang Zhang (HUAWEI)
+
+<!--
+The Kubernetes Special Interest Group (SIG) Release is proud to announce that we
+are digitally signing all release artifacts, and that this aspect of Kubernetes
+has now reached _beta_.
+-->
+Kubernetes 特别兴趣小组 SIG Release 自豪地宣布，我们正在对所有发布工件进行数字签名，并且
+Kubernetes 在这一方面现已达到 **Beta**。
+
+<!--
+Signing artifacts provides end users a chance to verify the integrity of the
+downloaded resource. It allows to mitigate man-in-the-middle attacks directly on
+the client side and therefore ensures the trustfulness of the remote serving the
+artifacts. The overall goal of out past work was to define the used tooling for
+signing all Kubernetes related artifacts as well as providing a standard signing
+process for related projects (for example for those in [kubernetes-sigs][k-sigs]).
+-->
+签名工件为终端用户提供了验证下载资源完整性的机会。
+它可以直接在客户端减轻中间人攻击，从而确保远程服务工件的可信度。
+过去工作的总体目标是定义用于对所有 Kubernetes 相关工件进行签名的工具，
+以及为相关项目（例如 [kubernetes-sigs][k-sigs] 中的项目）提供标准签名流程。
+
+[k-sigs]: https://github.com/kubernetes-sigs
+
+<!--
+We already signed all officially released container images (from Kubernetes v1.24 onwards).
+Image signing was alpha for v1.24 and v1.25. For v1.26, we've added all
+**binary artifacts** to the signing process as well! This means that now all
+[client, server and source tarballs][tarballs], [binary artifacts][binaries],
+[Software Bills of Material (SBOMs)][sboms] as well as the [build
+provenance][provenance] will be signed using [cosign][cosign]. Technically
+speaking, we now ship additional `*.sig` (signature) and `*.cert` (certificate)
+files side by side to the artifacts for verifying their integrity.
+-->
+我们已经对所有官方发布的容器镜像进行了签名（从 Kubernetes v1.24 开始）。
+在 V1.24 版本和 V1.25 版本中，镜像签名是 alpha 版本。
+在 V1.26 版本中，我们将所有的 **二进制工件** 也加入到了签名过程中！
```

Comment:
```suggestion
在 v1.24 版本和 v1.25 版本中，镜像签名是 alpha 版本。
在 v1.26 版本中，我们将所有的 **二进制工件** 也加入到了签名过程中！
```
The version notation for Kubernetes should be consistent with the original text.